google-cloud-spanner/src/main/java/com/google/cloud/spanner/MetricRegistryConstants.java

```diff
@@ -46,20 +55,20 @@
   // The Metric name and description
   static final String MAX_IN_USE_SESSIONS = "cloud.google.com/java/spanner/max_in_use_sessions";
   static final String MAX_ALLOWED_SESSIONS = "cloud.google.com/java/spanner/max_allowed_sessions";
-  static final String IN_USE_SESSIONS = "cloud.google.com/java/spanner/in_use_sessions";
```

Comment:
I'm worried for anyone relying on this metric already in their monitoring? What is the process for deprecating metrics in OpenCensus? Is there some monitoring to figure out if someone is exporting it?

Reply:
> What is the process for deprecating metrics in OpenCensus?

In OC, (usually) we first mark the existing API as deprecated and keep it for 18 months before removing it. I think the feature is fairly new and looks safe to remove it with a note in the Changelog.

> Is there some monitoring to figure out if someone is exporting it? 

I think it is feasible to see the ingested metrics in stackdriver, but I haven't done it personally.

> I'm worried for anyone relying on this metric already in their monitoring?

Another option is keep existing(`IN_USE_SESSIONS`) metric along with new metric. In this case, in_use_sessions data will be accounted/exported twice.

WDYT?

Reply:
Ok, I think you're right that this is relatively new. We will mark this in the Changelog.

Reply:
If you can include something about this in the PR description, that would be great. Then we can copy it in when doing the release.

Reply:
done